extensions/geo/Cargo.toml

```diff
@@ -27,6 +27,10 @@ vortex-error = { workspace = true }
 
 [dev-dependencies]
 vortex-scalar = { workspace = true }
+vortex-file = { workspace = true }
+geoarrow-geoparquet = { git = "https://github.com/geoarrow/geoarrow-rs.git", rev = "c241034f", features = [
+    "compression",
+] }
```

Comment:
By the way I've been considering a refactor to the geoparquet APIs so that users call upstream `parquet` APIs directly. https://github.com/geoarrow/geoarrow-rs/pull/1089